app/javascript/elm/src/Main.elm

```diff
@@ -467,6 +478,7 @@ viewHeatMapSlider heatMapThresholds sensorUnit =
         [ div [ class "heatmap-input" ] [ viewHeatMapInput "min" threshold1 sensorUnit UpdateHeatMapMinimum ]
         , div [ id "heatmap" ] []
         , div [ class "heatmap-input" ] [ viewHeatMapInput "max" threshold5 sensorUnit UpdateHeatMapMaximum ]
+        , button [ Events.onClick ResetHeatMapToDefaults ] [ text "D" ]
```

Comment:
couldn't find that button :D why not sth like "Reset heatmap"?

Reply:
aaa... https://llp.kanbanery.com/projects/8040/board/tasks/2403287/attachments
:)